beaconChain/src/helpers/stateTransitionHelpers.ts

```diff
@@ -197,21 +198,22 @@ export function split<T>(values: T[], splitCount: int): T[][] {
   return array;
 }
 
-/**
- * Helper function for readability.
- * @param {int} minval
- * @param {int} maxval
- * @param {int} x
- * @returns {int}
- */
-export function clamp(minval: int, maxval: int, x: int): int {
-  if (x <= minval) {
-    return minval;
-  } else if (x >= maxval) {
-    return maxval;
-  }
-  return x;
-}
+// TODO: Remove; no longer in spec
+// /**
+//  * Helper function for readability.
+//  * @param {int} minval
+//  * @param {int} maxval
+//  * @param {int} x
+//  * @returns {int}
+//  */
+// export function clamp(minval: int, maxval: int, x: int): int {
+//   if (x <= minval) {
+//     return minval;
+//   } else if (x >= maxval) {
+//     return maxval;
+//   }
+//   return x;
+// }
```

Comment:
probably best to remove entirely, same with clamp tests

Reply:
+1